cypress.config.ts

```diff
@@ -0,0 +1,18 @@
+import { defineConfig } from "cypress";
```

Comment:
Cypress 10 introduced a different config and file names. These changes are related to that.